wofi-pass

```diff
@@ -104,24 +104,28 @@ EOF
 }
 
 get_config_file () {
-    [ ! "$XDG_CONFIG_HOME" ] && export XDG_CONFIG_HOME="$HOME/.config"
-    configs=("$XDG_CONFIG_HOME/wofi-pass/config"
-             "/etc/wofi-pass.conf")
+    local config_base_dir="${XDG_CONFIG_HOME:-"${HOME}/.config"}"
+    readonly config_base_dir
+
+    config_files=("${config_base_dir}/wofi-pass/config"
+                  "/etc/wofi-pass.conf")
 
```

Comment:
One last point. I think it is a good idea to add the `WOFI_PASS_CONFIG` to `config_files` and add the `-n` stuff in the `if` expression again (line 115).
Because without it, it is not possible to define your own wofi-pass configuration file path.

You mentioned that there is problem with `set -u`, I guess this should work:
```bash
local config_files=("${WOFI_PASS_CONFIG:-""}"
                              "${XDG_CONFIG_HOME:-"${HOME}./config"}"
                              "/etc/wofi-pass.conf")
```
What do you thing?